### PR TITLE
feat: improve design of about page

### PR DESCRIPTION
**Pull request by shootermv**
# Description
add some improvements to about page styling

## screenshots
image

### CODE REVIEW

**Comment by NoamGaash** [comment]
it looks so much better! :clap:  :clap:  :clap:

File: src/pages/about/About.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+ul {
+    list-style: none;
+    padding: 0;
+}
+
+li img {
+    width: 5%;
```
Comment (NoamGaash) [inline src/pages/about/About.scss:7]: did you check that on different screen sizes?

**Review by NoamGaash — APPROVED** [review]
perfect!

File: src/pages/about/About.scss (hunk 1 above)
Comment (shootermv) [reply src/pages/about/About.scss:7]: is there something wrong you can see now?

Comment (shootermv) [reply src/pages/about/About.scss]: changed img width to `1.9em`
image

Comment (NoamGaash) [reply src/pages/about/About.scss]: I didn't check how it looks, only the code and screenshot.
consider adding `margin-right` and bullets to list

**Pull request merged**